.travis.yml

```diff
@@ -7,40 +7,10 @@ env:
   - COMMIT_AUTHOR_EMAIL=skynet@open.qa
 matrix:
   include:
-    - name: "unit- and integration tests"
-      env: FULLSTACK=0 UITESTS=0
-    - name: "UI tests"
-      env: FULLSTACK=0 UITESTS=1
-    - name: "fullstack test"
-      env: FULLSTACK=1
-    - name: "scheduler fullstack test"
-      env: SCHEDULER_FULLSTACK=1
-    - name: "developer fullstack test"
-      env: DEVELOPER_FULLSTACK=1
-    - name: "documentation generation job"
-      env: BUILD_DOCS=1
```

Comment:
you still keep the file but remove all content from "include:"? Does that even work?

Reply:
Yes, the idea is that we must test documentation generation in 15.1, because it looks quite different and I am not sure how to test it reliably.
So for now Travis still generates documentation as before and Circleci pushes generated documentation into hithub.com/openqabot/openQA:gh-pages
Once we reviewed new documentation generation - we can disable Travis and then let CircleCI to push into upstream (or just let official gh-pages to be in openqabot, which is more secure (because CI doesn't push to upstream) and will be better for size of upstream repo).